routingpy/routers/opentripplanner_v2.py

```diff
@@ -198,6 +203,7 @@ def _parse_legs(self, legs):
             geometry.extend(points)
             distance += int(leg["distance"])
 
+        geometry = [coord[::-1] for coord in geometry]
```

Comment:
will this be lng/lat or lat/lng?

Reply:
Yes, it's lon, lat as specified in the Direction class.

Reply:
Can we fix this 3 lines further up? I.e. `geometry.extend(list(reversed(points))`

Reply:
Done